dbms/src/Storages/KVStore/TiKVHelpers/DecodedLockCFValue.h

```diff
@@ -39,8 +39,9 @@ struct DecodedLockCFValue : boost::noncopyable
     UInt64 min_commit_ts{0};
     std::string_view secondaries;
     std::string_view primary_lock;
+    bool is_txn_file{false};
```

Comment:
Better to keep same format with `bool use_async_commit{0};`

Reply:
```suggestion
    bool is_txn_file{0};
```

Reply:
OK,  i changed back /cc @JaySon-Huang